### PR TITLE
ObjectGraphService#inject(Scope, Object)

### PR DESCRIPTION
**Pull request by rjrjr**
We have similar parallel Scope/Context method pairs throughout the services.

### CODE REVIEW

**Comment by rjrjr** [comment]
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Mortar (Parent) .................................... SUCCESS [  1.724 s]
[INFO] Mortar ............................................. SUCCESS [  8.970 s]
[INFO] Dagger1 Support .................................... SUCCESS [  4.810 s]
[INFO] Dagger2 Support .................................... SUCCESS [  0.535 s]
[INFO] Hello Mortar ....................................... SUCCESS [ 10.327 s]
[INFO] Hello Dagger2 ...................................... SUCCESS [  6.528 s]
[INFO] Mortar Sample ...................................... SUCCESS [ 10.719 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 48.023 s
[INFO] Finished at: 2015-03-06T13:02:04-08:00
[INFO] Final Memory: 39M/311M
[INFO] ------------------------------------------------------------------------
```

File: dagger1support/src/main/java/mortar/dagger1support/ObjectGraphService.java [hunk 1]
```diff
@@ -41,6 +41,15 @@ public static void inject(Context context, Object object) {
     getObjectGraph(context).inject(object);
   }
 
+  /**
+   * A convenience wrapper for {@link ObjectGraphService#getObjectGraph} to simplify dynamic
+   * injection, typically for {@link Activity} and {@link android.view.View} instances that must be
+   * instantiated by Android.
+   */
+  public static void inject(MortarScope scope, Object object) {
```
Comment (loganj) [inline dagger1support/src/main/java/mortar/dagger1support/ObjectGraphService.java]: Nit: Maybe rename `object` to `target` or something.

**Comment by loganj** [comment]
LG

File: dagger1support/src/main/java/mortar/dagger1support/ObjectGraphService.java (hunk 1 above)
Comment (pyricau) [inline dagger1support/src/main/java/mortar/dagger1support/ObjectGraphService.java]: Should be `instance`, as in Dagger`:

```
  /**
   * Injects the members of {@code instance}, including injectable members
   * inherited from its supertypes.
   *
   * @throws IllegalArgumentException if the runtime type of {@code instance} is
   *     not one of this object graph's {@link Module#injects injectable types}.
   */
  public abstract <T> T inject(T instance);
```

**Pull request merged**